people/our-values.md

```diff
@@ -0,0 +1,125 @@
+# Our Values
+
+Here at t'shed, we have 5 core values to help support our vision, shape our
```

Comment:
We switch between *t'Shed*, *The Shed* and *The Data Shed* throughout—worth using one for consistency?

(Whichever way, this should be a capital *S*.)

Reply:
Gone for The Data Shed since you've mentioned this is available externally